rubygem-foreman-tasks-core/rubygem-foreman-tasks-core.spec

```diff
@@ -0,0 +1,62 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman-tasks-core
+
+Summary: Code used both at Forman and Foreman proxy regarding tasks
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.1
+Release: 1%{?foremandist}%{?dist}
+Group: Development/Languages
+License: GPLv3
+URL: https://github.com/theforeman/foreman-tasks
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
```

Comment:
ditto, Source0 should be HTTP for now. The SHA256 sum is stored by git-annex in the repo so the downloaded file is still verified.